src/core/on-chain/index.ts

```diff
@@ -84,24 +58,22 @@ export const OnChainMixin = (superclass) =>
 
       let fee
 
-      // FIXME: legacy. is this still necessary?
-      const defaultAmount = 300000
-
       if (payeeUser) {
         fee = 0
       } else {
         if (amount && amount < this.config.dustThreshold) {
           throw new DustAmountError(undefined, { logger: this.logger })
         }
-
-        // FIXME there is a transition if a node get offline for which the fee could be wrong
-        // if send by a new node in the meantime. (low probability and low side effect)
-        const { lnd } = getActiveOnchainLnd()
-
-        const sendTo = [{ address, tokens: amount ?? defaultAmount }]
         try {
-          ;({ fee } = await getChainFeeEstimate({ lnd, send_to: sendTo }))
+          // (numeric, optional) estimate fee rate in BTC/kB (only present if no errors were encountered)
+          const result = await this.bitcoindClient.estimateSmartFee({
+            conf_target: 1,
+          }) // TODO conf_target
+          const feerate = result.feerate
+          // 1 BTC/kB = 100000 satoshis/byte
+          fee = 100000 * feerate // TODO not sure about this: is fee by size, not a total fee
```

Comment:
Confirmed that it is different from lightning's getChainFeeEstimate, which does return a total [fee](https://github.com/alexbosworth/ln-service#getchainfeeestimate).